src/BIMDataComponents/BIMDataSelect/BIMDataSelectMulti.vue

```diff
@@ -137,9 +137,9 @@ export default {
         return this.options;
       } else {
         const lowerCaseSearchText = this.searchText.toLowerCase();
-        return this.options.filter(option => {
-          return option.toLowerCase().includes(lowerCaseSearchText);
-        });
+        return this.options.filter(option =>
+          option.toLowerCase().includes(lowerCaseSearchText),
```

Comment:
```suggestion
          option.toLowerCase().includes(lowerCaseSearchText)
```

Reply:
Curiosity question: is this comma a https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Comma_operator in this case?